src/Features/CSharp/Portable/Completion/CompletionProviders/NamedParameterCompletionProvider.cs

```diff
@@ -254,12 +254,7 @@ int IEqualityComparer<IParameterSymbol>.GetHashCode(IParameterSymbol obj)
 
         protected override Task<TextChange?> GetTextChangeAsync(CompletionItem selectedItem, char? ch, CancellationToken cancellationToken)
         {
-            return Task.FromResult<TextChange?>(new TextChange(
-                selectedItem.Span,
-                // Do not insert colon on <Tab> so that user can complete out a variable name that does not currently exist.
-                // ch == null is to support the old completion only.
-                // Do not insert an extra colon if colon has been explicitly typed.
-                (ch == null || ch == '\t' || ch == ':') ? selectedItem.DisplayText : selectedItem.GetEntireDisplayText()));
+            return Task.FromResult<TextChange?>(new TextChange(selectedItem.Span, selectedItem.DisplayText));
```

Comment:
Ok.  while this is a change in behavior, *think* it's going to give preferable behavior. We should keep our eyes and ears open for people who may feel this is some sort of regression in how they like to type their code...